src/network/netdev/tunnel.c

```diff
@@ -422,6 +441,13 @@ static int netdev_tunnel_verify(NetDev *netdev, const char *filename) {
                 return -EINVAL;
         }
 
+        if (t->fou_tunnel) {
+                if (t->fou_destination_port <= 0) {
```

Comment:
These two `if` statements can be merged.

Reply:
yes